assets/src/edit-story/components/panels/textStyle/getClosestFontWeight.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Returns a number representing closest available font weight to previously selected.
+ *
+ * @param {number} existingWeight previously selected font weight value to get as close to as possible, defaults to 400 (normal).
+ * @param {Array<string>} availableWeights available font weights for newly selected font.
+ * @return {number} Closest font weight available for given font.
+ */
+
+const DEFAULT_FONT_WEIGHT = 400;
+
+function getClosestFontWeight(
```

Comment:
What bugs me here is that we once had this already in #927 😞  Glad we're adding tests now to make sure this doesn't regress again! 👍

Reply:
ah that's a bummer, i didn't know that! I do kind of get why that was removed/changed though - when we have text elements that have multiple weights applied to them the weight of the text element has a legit value of 'multiple' whereas the bug that we were seeing here was that the placeholder was coming back because no value could be matched.